packages/testing/src/api/index.ts

```diff
@@ -22,6 +22,8 @@ import * as grpc from 'grpc';
 import { resolve } from 'path';
 import * as fs from 'fs';
 
+export { database, firestore } from 'firebase';
```

Comment:
why is this here?

Reply:
To expose the functions inside of these submodules without having to explicitly import them (for our clients)